service/src/test/java/tds/student/services/impl/RtsServiceImplIntegrationTests.java

```diff
@@ -0,0 +1,97 @@
+package tds.student.services.impl;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.support.AbstractLobCreatingPreparedStatementCallback;
+import org.springframework.jdbc.support.lob.DefaultLobHandler;
+import org.springframework.jdbc.support.lob.LobCreator;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Optional;
+
+import tds.dll.common.rtspackage.common.exception.RtsPackageWriterException;
+import tds.dll.common.rtspackage.student.StudentPackageWriter;
+import tds.student.RtsStudentPackageAttribute;
+import tds.student.services.RtsService;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@Transactional
+public class RtsServiceImplIntegrationTests {
+    @Autowired
+    private DataSource dataSource;
+    private JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    private RtsService rtsService;
+
+    private byte[] studentPackageBytes;
+
+    @Before
+    public void setUp() throws Exception {
+        jdbcTemplate = new JdbcTemplate(dataSource);
+        InputStream inputStream = getClass().getClassLoader().getResourceAsStream("studentPackage.xml");
+        String studentPackage = IOUtils.toString(inputStream);
+
+        StudentPackageWriter writer = new StudentPackageWriter();
+        writer.writeObject(studentPackage);
+        studentPackageBytes = IOUtils.toByteArray(writer.getInputStream());
+    }
+
+    @Test
+    public void shouldReadPackage() throws IOException, RtsPackageWriterException {
+        insertData();
+
+        Optional<RtsStudentPackageAttribute> maybeRtsAttribute = rtsService.findRtsStudentPackageAttribute("SBAC_PT", 1, "LglFNm");
+
+        assertThat(maybeRtsAttribute).isPresent();
+        assertThat(maybeRtsAttribute.get().getName()).isEqualTo("LglFNm");
+        assertThat(maybeRtsAttribute.get().getValue()).isEqualTo("ASL");
+    }
+
```

Comment:
As far as coverage I think it would be nice to add a test where the casing of the attribute doesn't match to make sure it returns as expected, whatever that might be.  Not sure what that is going to be actually?